### ISSUE TITLE
If balance < 50$ on mech turk account, ping Chris or Anders

### ISSUE THREAD

**Issue by anderspeders** [NRGI/resourcecontracts.org#549]
@byndcivilization suggested that there might be an api method to check balance? 

Suggest that a notification is sent to check when we're getting low.

**Comment by anjesh** [comment]
@byndcivilization are you using amazon payment in Mturk account? I think we talked about this. Perhaps it has some notifications in itself, otherwise we will see their APIs to get this notification.

**Comment by anjesh** [comment]
Will be addressed here. https://github.com/NRGI/resourcecontracts.org/issues/632

**Comment by anjesh** [comment]
Need to show the amount remaining in the Mturk contract listing page and also trigger email in case of Mturk balance gets lower than 10$,

**Comment by anjesh** [comment]
The email is sent to the group mail resourcecontracts@yipl.com.np and also shown in the dashboard.

**Comment by anjesh** [comment]
The emails are coming to resourcecontracts@yipl.com.np when the balance in lower than 50$ with the following example content.

```
MTurk balance is getting low. Please top-up amount on time

Current Available Balance: $13.24
```
